tools/test-generator/src/main/java/com/consol/citrus/generate/provider/http/SendHttpRequestActionProvider.java

```diff
@@ -60,7 +62,7 @@ public SendRequestModel getAction(String endpoint, HttpMessage message) {
 
         requestType.setHeaders(requestHeaders);
 
-        if (!CollectionUtils.isEmpty(message.getQueryParams())) {
+        if (!message.getQueryParams().isEmpty()) {
```

Comment:
This could cause a null pointer, if `message.getQueryParams()` returns `null`. If you change the signature of `message.getQueryParams()` back to a `Map<K, V>` implementation, `CollectionUtils.IsEmpty()` will be usable again, which includes a null check.